CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# dbt_quickbooks v0.13.0
+[PR #124](https://github.com/fivetran/dbt_quickbooks/pull/124) includes the following updates:
+
+## 🚨 Breaking Changes 🚨:
+- Updates the [int_quickbooks__invoice_join](https://github.com/fivetran/dbt_quickbooks/blob/main/models/intermediate/int_quickbooks__invoice_join.sql) and downstream [quickbooks__ap_ar_enhanced](https://github.com/fivetran/dbt_quickbooks/blob/main/models/quickbooks__ap_ar_enhanced.sql) models to include and require the `using_payments` config. Previously, these models would fail if the `payment` or the `payment_line` source tables did not exist.
+- Corrects the misspelled `customer_vendor_webiste` to `customer_vendor_website` in `quickbooks__ap_ar_enhanced`.
+
+## Bug Fixes
+- Updates the [quickbooks__profit_and_loss](https://github.com/fivetran/dbt_quickbooks/blob/main/models/quickbooks__profit_and_loss.sql) and [quickbooks__balance_sheet](https://github.com/fivetran/dbt_quickbooks/blob/main/models/quickbooks__balance_sheet.sql) models to include both `period_first_day` and `period_last_day`. This allows users to have greater flexibility in choosing which date to aggregate records upon.
```

Comment:
is this a bug fix or feature update?

Reply:
also are we intending on deprecating `calendar_date`? if so, maybe we should mention it here (and maybe in `calendar_date`'s yml description) and recommend using one of the new fields instead

Reply:
Thanks @fivetran-jamie , moved that to its own feature update section and added the deprecation notice for `calendar_date`:

"This is slated to be deprecated, and the fields `period_first_day` and `period_last_day` are both offered as replacements, depending on how your company performs their financial reporting."

Reply:
that sounds great! do you have the feature_update section added locally?